### DISCUSSION TITLE
为什么defineProps不能识别vant导入的类型

### DISCUSSION THREAD

**Discussion by zzydannyer** [youzan/vant#12982] [Q&A]
`  import { UploaderProps } from 'vant';
  const props = withDefaults(defineProps<UploaderProps>(), {});
`
报错：[plugin:vite:vue] [@vue/compiler-sfc] Unresolvable type reference or unsupported built-in utility type

 "vue": "^3.4.31",
 "vant": "^4.9.1",
 "vite": "5.3.2",

**Comment by bohecola** [comment]
Has it been resolved ?
